index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width" />
+	<link href="https://fonts.googleapis.com/css?family=Gelasio:400,500,600&display=swap" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Poppins:400,500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="normalize.css" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+	<title>Mark &amp; Markup &vert; Full stack developer portfolio</title>
+</head>
+
+<body>
+	<header>
+		<div class="header-top">
+			<img src="./images/mamu-logo2.png" alt="Mark and Markup Logo" class="logo" />
+			<h1>Mark<span class="accent-color">&amp;</span>Markup</h1>
+			<img src="./images/hamburger-icon.png" alt="Menu Icon" id="burger-menu"/>
+		</div>
+		
+		<nav>
+
+			<div class="nav-links">
+				<ul>
+					<li><a href="#about">About</a></li>
+					<li><a href="#portfolio">Portfolio</a></li>
+					<li><a href="#">Contact</a></li>
+				</ul>
+			</div>
+		</nav>
+	</header>
+	
+	<section class="showcase">
+		<div class="hero-caption">
+			<h1>Mark Eyles</h1>
+			<h3>Full stack developer portfolio</h3>
+		</div>
+		<a href="#" class="a-box">View this project ></a>
+	</section>
+	
+	<section class="about">
+		<a href="#" id="about" class="a-section-anchor">&nbsp;</a>	
+		<div class="center-container">
+			<a href="#about" class="a-section-title"><h2>About</h2></a>
+			<div class="about-content">
+				<img src="./images/about-me-square.jpg" alt="Mark Eyles leaning on a blue container" class="headshot" />
+				<p><span class="accent-largetext accent-color accent-serif">Hi there!</span> I'm Mark - a Bristol-based full stack developer with big ideas and a knack for problem solving.</p>
+				<p>With a background as a creative producer covering all sorts of music and video projects, I've now turned my hand to the fascinating world of coding. A recent graduate from Mayden Academy in Bath, I'm well versed in a variety of front- and back-end technologies including <span class="accent-color">html</span>, <span class="accent-color">css</span>, <span class="accent-color">javascript</span>, <span class="accent-color">PHP</span>, <span class="accent-color">node.js</span> and <span class="accent-color">React</span>.</p>
+				<p>Take a look at some of my projects below to see what I can do, and feel free to get in touch if you'd like to work together! Bonus points if you're a fan of good food, squash (the sport), or scuba diving.</p>
+			</div>
+		</div>
+	</section>
+	
+	<section class="portfolio">
+		<a href="#" id="portfolio" class="a-section-anchor">&nbsp;</a>
+		<div class="center-container">
+			<a href="#portfolio" class="a-section-title"><h2>Project Portfolio</h2></a>
+			<p class="touchscreen-info">Tap an image for a project summary</p>
+			<div class="portfolio-content">
```

Comment:
same again with center-container - could probably strip out one extra layer of div